app/jobs/decidim/direct_verifications/register_users_job.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "decidim/direct_verifications/instrumenter"
+
+module Decidim
+  module DirectVerifications
+    class FakeSession < Hash
+    end
+
+    class RegisterUsersJob < ApplicationJob
+      queue_as :default
+
+      def perform(userslist, organization_id, user_id)
+        @userslist = userslist
+        @organization = Organization.find_by(id: organization_id)
+        @current_user = User.find_by(id: user_id)
```

Comment:
apparently Active Job can do this for us https://edgeguides.rubyonrails.org/active_job_basics.html#globalid